### ISSUE TITLE
No highlights applied - Please help debugging

### ISSUE THREAD

**Issue by weilbith** [LiadOz/nvim-dap-repl-highlights#9]
Hey 👋🏾 

I think the idea of the plugin is great and I would love to use it. Unfortunately I'm unable to make it work. 🙈 

So if I run `:TSInstallInfo`, I can see `dap_repl` as installed. If I run `:lua =vim.treesitter.inspect_language('dap_repl')`, I get the following output:

```lua
{
  _abi_version = 14,
  fields = {},
  symbols = { { "\n", false },
    [3] = { "prompt_tag", true },
    [4] = { "continue_prompt_tag", true },
    [5] = { "source_file", true },
    [6] = { "entry", true },
    [7] = { "prompt", true },
    [8] = { "user_input", true },
    [10] = { "user_input_statement", true },
    [11] = { "user_input_content", true },
    [12] = { "output", true },
    [13] = { "output_content", true }
  }
}
```

Anyhow, if I start a debugging session and have the `dap-repl` buffer opened, I don't get any highlighting. How can I investigate further why the highlights don't apply?
I'm using a Rust project with `codelldb`.

Thank you very much! 🙏🏾

**Comment by LiadOz** [comment]
Hi, Have you tried using the setup_highlights function detailed in the README? this manually triggers the highlights for the selected language. If that function works, it is likely the issue is in your configuration

**Comment by weilbith** [comment]
Hmm. So I run this, but it doesn't change anything.
Though, by now I'm a little confused about the `dap-repl`. 
So if I type `p foo + 1 > 0` I don't get any highlights. Maybe I should try it with some other languages and debuggers. 
Do you use it with Rust?

**Comment by weilbith** [comment]
Closing because I lack in knowledge.

**Comment by weilbith** [comment]
I tried it now with Python because it has full proper REPL support. But still not working. Am confused.

**Comment by LiadOz** [comment]
Can you give the output of the `:version` command and your configuration?

**Comment by weilbith** [comment]
My version is: `NVIM v0.10.0-dev-1913041`

And my configuration is basically this:
```
require('packer').use({
  'LiadOz/nvim-dap-repl-highlights',
  requires = 'nvim-treesitter/nvim-treesitter',
  ft = 'dap-repl',
  config = function()
    require('nvim-dap-repl-highlights').setup()
  end,
  run = function()
    require('nvim-dap-repl-highlights').setup()
    print('yeah')
    vim.cmd('TSInstall dap_repl')
  end,
})
```

**Comment by LiadOz** [comment]
Can you try removing the `ft` line?

**Comment by LiadOz** [comment]
You can do the following to check if the issue is in the parsing side:
1. Remove the call for `require('nvim-dap-repl-highlights').setup()` in the config.
2. Open a new buffer with name scratch (without any extension)
3. Paste the following as content -
```
dap> def hi():
dap>     pass
```
4. Run `:require('nvim-dap-repl-highlights').setup()` then `:lua require('nvim-dap-repl-highlights').setup_highlights('python')` in the scratch buffer.
5. The scratch buffer should now have highlights.

This should trigger the highlighting manually and will rule out configuration issues.

**Comment by weilbith** [comment]
Thanks for the instructions. So I just opened NeoVim and executed `:edit scratch` and followed your instructions.
Indeed something happened: the `dap>` texts became grey. But the rest remains white as normal. Is that the expected behavior? I would have expected the function definition to be highlighted.
In the screenshot below I have the buffer as instructed by you on the left. On the right is a buffer I set the filetype to Python and removed the `dap>` parts of the code.
![image](https://github.com/LiadOz/nvim-dap-repl-highlights/assets/12543647/1fb13944-1f53-41b2-b5a1-58a1680f83a9)
